roles/neutron/tasks/juno.yml

```diff
@@ -260,7 +287,9 @@
   tags: neutron
 
 - name: disable neutron services
```

Comment:
I'd like to get a better understanding of what we are doing here with services.  I see there are different conditionals, but it seems there is redundancy.  At a minimum, we should update the task name to describe more fully what/why we are performing the task (unique task names).